tests/unit/test_metric_parsing.py

```diff
@@ -186,6 +190,7 @@
 }
 
 MENDIX_8_JAVA_11_STATS_PARALLELGC = {
+    "code": 0,
```

Comment:
What is the reason for adding "code" here to this dict? Originally this dict was something that I copied from the output of the calls direct to the runtime, so I'm curious why we are changing it :)

Reply:
Oh ok. I thought we had missed the 'code' keys in Java 11, as I see it in the older Java version dicts.
But I just confirmed, it indeed is missing from the runtime stats.

I'll remove those.